product_serial/wizard/prodlot_wizard.py

```diff
@@ -22,116 +22,150 @@
 #
 ##############################################################################
 
-from openerp.osv import orm, fields
-from openerp.tools.translate import _
-
-
-class stock_picking_prodlot_selection_wizard(orm.TransientModel):
-    _name = 'stock.picking.prodlot.selection'
-
-    _columns = {
-        'product_id': fields.many2one(
-            'product.product', 'Product', required=True),
-        'prefix': fields.char('Prefix', size=256),
-        'suffix': fields.char('Suffix', size=256),
-        'first_number': fields.char('First Number', size=256, required=True),
-        'last_number': fields.char('Last Number', size=256, required=True),
-        'create_prodlots': fields.boolean('Create New Serial Numbers'),
-    }
-
-    _defaults = {
-        'create_prodlots': False,
-    }
-
-    def select_or_create_prodlots(self, cr, uid, ids, context=None):
-        if context is None:
-            context = {}
-        if not ids:
-            return {}
-        if 'active_id' not in context:
-            return {}
-
-        prodlot_obj = self.pool['stock.production.lot']
-        record = self.browse(cr, uid, ids[0], context)
-        prefix = record.prefix or ''
-        suffix = record.suffix or ''
+from openerp import models, fields, api, _
+from openerp.exceptions import Warning as UserError
+import base64
+
+
+class StockProdlotSelection(models.TransientModel):
+    _name = 'stock.prodlot.selection'
+    _description = "Select or Create Production Lots"
+
+    product_id = fields.Many2one(
+        'product.product', string='Product', required=True)
+    prefix = fields.Char(string='Prefix')
+    suffix = fields.Char(string='Suffix')
+    first_number = fields.Char(string='First Number')
+    last_number = fields.Char(string='Last Number')
+    prodlot_file = fields.Binary(
+        string='Serial Numbers File',
+        help="The serial numbers file should be a text file with one line "
```

Comment:
s/text/csv(?)
You should detail the file structure (here or in README) or provide an example in the module.